core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3376,6 +3406,110 @@ private void setAttributeFromEntry(SetAttributeEntry entry)
     return mBlockMaster.getWorkerInfoList();
   }
 
+  @Override
+  public void updateUfsMode(String ufsPath, UnderFileSystem.UfsMode ufsMode)
+      throws InvalidPathException {
+    try (JournalContext journalContext = createJournalContext()) {
```

Comment:
Should there be an audit log entry as well, since this is like an admin command?